9.2/run-postgresql.sh

```diff
@@ -27,12 +28,34 @@ function usage() {
 	echo "  POSTGRESQL_DATABASE (regex: '$psql_identifier_regex')"
 	echo "Optional:"
 	echo "  POSTGRESQL_ADMIN_PASSWORD (regex: '$psql_password_regex')"
+	echo "  POSTGRESQL_REPLICA (true or false)"
+	echo "  POSTGRESQL_MASTER (host or ip address of master)"
 	echo "Settings:"
 	echo "  POSTGRESQL_MAX_CONNECTIONS (default: 100)"
 	echo "  POSTGRESQL_SHARED_BUFFERS (default: 32MB)"
 	exit 1
 }
 
+# Poll until postgres responds to our ping.
+function wait_for_pg() {
+
+while [ true ]; do
+#        pg_isready -h $POSTGRESQL_MASTER -U $POSTGRESQL_USER -p 5432 -d $POSTGRESQL_DATABASE
+	psql -h $POSTGRESQL_MASTER -U $POSTGRESQL_USER -p 5432 -c "select now()"  $POSTGRESQL_DATABASE
+        export RC=$?
+
+        echo "master connectivity test return code " $RC
+        if [ $RC = "0" ] ; then
+                echo "connection reached to master"
+                return 1
+        fi
+        echo "waiting for master pg to start..."
+        sleep 5
+done
```

Comment:
These should be tabs. I'll probably want to change indentation to two spaces anyway, so not a big deal.